mail_edit/static/src/js/tour.js

```diff
@@ -0,0 +1,56 @@
+odoo.define('mail_edit.tour', function (require) {
+"use strict";
+
+var core = require('web.core');
+var tour = require('web_tour.tour');
+
+var _t = core._t;
+
+tour.register('mail_edit_tour', {
+    url: "/web",
+}, [tour.STEPS.SHOW_APPS_MENU_ITEM, {
+    trigger: '.o_app[data-menu-xmlid="mail.menu_root_discuss"]',
+    content: _t('Open Apps -> Discuss menu'),
+    position: 'right',
+    edition: 'community',
+}, {
+    trigger: ".o_mail_discuss_item[data-thread-id=1]",
+    auto: true,
+    in_modal: false,
+}, {
+    trigger: '.o_mail_discuss .o_composer_text_field',
+    content: _t("Add some text message to the composer chat box"),
+    position: "top",
+    width: 350,
+    run: function (actions) {
+        actions.text("Sample Message to be edited & Deleted!", this.$anchor);
+    },
+}, {
+    trigger: '.o_mail_discuss .o_thread_composer .o_composer_button_send',
+    content: _t("Post your message on the thread"),
+    position: "top",
```

Comment:
The test now runs, but looking at the HTML output it gives, it seems that the screen that opens after this step does not have an element with `o_mail_info`, and thus also no `.o_thread_edit_icon .o_mail_edit`, and so the next step fails.